ci/deps/azure-37-numpydev.yaml

```diff
@@ -3,20 +3,19 @@ channels:
   - defaults
 dependencies:
   - python=3.7.*
-  - pytz
-  - Cython>=0.29.13
-  # universal
-  # pytest < 5 until defaults has pytest-xdist>=1.29.0
-  - pytest>=4.0.2,<5.0
-  - pytest-xdist
+  # tools
+  - cython=0.29.13
+  - pytest
+  - pytest-xdist>=1.21
   - pytest-mock
   - hypothesis>=3.58.0
+  - pytest-azurepipelines
```

Comment:
does this have a min/max pin?

Reply:
I think there is just one version or two of this. I don't think we care much (not even sure if it adds any value)